content/doc/best-practices/tips_and_tricks.md

```diff
@@ -0,0 +1,226 @@
+
+| Key | Description |
+| --- | ----------- |
+| `?` | Show keyboard shortcuts help |
+| `a` | Jump to application activity |
+| `d` | Jump to application domains |
+| `e` | Jump to application environment variables |
+| `i` | Jump to application / add-on information |
+| `l` | Jump to application / add-on logs |
+| `m` | Jump to Application metrics |
+| `o` | Jump to application overview |
+| `s` | Jump to application scalability |
+
+### Launch applications as one-time tasks
+
+One of our missions is to keep your applications and services up and running, providing logs, metrics, scaling features, and more. But sometimes, you need to launch an application to execute a one-time task. On Clever Cloud, there is an option for that.
+
+Just define an application as a task in its `Information` panel or at creation with Clever Tools (`clever create --type TYPE --task "command to execute"`). The application will start, fetch its dependencies and build as usual, but then just execute the command and stop.
+
+- [Learn more about Clever Tasks](/doc/develop/tasks/)
+
+### Patch update or redeploy a database
+
+When Clever Cloud release a new major/minor version of a database add-on, you can use the migration tool from the Console to upgrade it without downtime. It also allows to move the add-on to another zone. But sometimes, you just need the latest patch version or to redeploy your database. In such case, migrate it to the exact same version, plan, zone and launch the migration process.
+
+- [Learn more about Clever Cloud database add-ons](/doc/addons/)
+- [Learn more about Clever Cloud database migration](/doc/administrate/database-migration/)
+
+## Clever Tools
+
+[Installing our CLI](https://github.com/CleverCloud/clever-tools/blob/58032c8c3ffdd56eb9da3a847d1fd2485eb489b9/docs/setup-systems.md#how-to-install-clever-tools) is as easy as `npm i -g clever-tools`. Then, just run `clever login` and you're ready to go.
+
+### SSH in any application without effort
+
+Clever Cloud Platform-as-a-Service (PaaS) provides immutable infrastructure. While SSH access isn't necessary nor recommended, Clever Tools provides a `clever ssh` command to SSH into instances, helpful for debugging purposes. You can pass a key to it:
+
+```bash
+clever ssh
+clever ssh --app app_id_or_name
+clever ssh -a app_id_or_name -i ~/.ssh/id_ed25519
+```
+
+If you need to diagnose a problem, you can set the `CC_TROUBLESHOOT` environment variable to `true`. Once started, the application will stay up even if there is an error (up to 1 hour).
+
+- [Lean more about SSH on Clever Cloud](/doc/account/ssh-keys-management/#configure-your-ssh-agent)
+
+### Diagnose domains DNS configuration of your applications
+
+Clever Tools provides a `domain diag` command to check the DNS configuration of domains linked to your applications. It doesn't cover cases such as when you use a CDN, but it will tell you if everything is OK or if you need to fix something.
```

Comment:
Also, I feel like the `domain overview` command would be a good fit for this page, WDYT? It's only useful for big orgs but we've just added it and it can definitely come in handy for them :eyes:

Reply:
It's more a CLI feature to list add-ons, quite useful, but we need to avoid duplicate the whole CLI documentation here :p I take this as a suggestion for next iterations.